### PR TITLE
Added functions from Work to centripetal force

### PR DESCRIPTION
**Pull request by SagarJC**
Also added rotational mechanics function - angular displacement, angular velocity, angular acceleration, moment of inertia, angular momentum, angular kinetic energy, centripetal accceleration and torque. imported all functions from numpy and specified the value of gravitational constant which might be used in future.

### CODE REVIEW

File: naxxatrapy/naxxatrapy.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+G = 6.674 × 10**(-11)
+from numpy import *
```
Comment (jee1mr) [inline naxxatrapy/naxxatrapy.py]: Import statement should always be at the top of the file

**Comment by jee1mr** [comment]
@SagarJC the docstring should start after def(): Can you remove the extra blank lines in between in all your functions?

File: naxxatrapy/naxxatrapy.py (hunk 1 above)
Comment (SagarJC) [reply naxxatrapy/naxxatrapy.py]: Ok will do the changes.

Comment (jee1mr) [reply naxxatrapy/naxxatrapy.py]: @SagarJC Remove numpy. Use "math"

**Review by jee1mr — APPROVED** [review]

**Pull request merged**